docs/static/releasenotes.asciidoc

```diff
@@ -1,17 +1,25 @@
 [[releasenotes]]
 == Release Notes
 
+coming[8.0.0]
+
 This section summarizes the changes in the following releases:
 
-* <<logstash-7-0-0-alpha1,Logstash 7.0.0-alpha1>>
-* <<logstash-7-0-0-alpha2,Logstash 7.0.0-alpha2>>
 
-[[logstash-7-0-0-alpha1]]
-=== Logstash 7.0.0-alpha1 Release Notes
+//* <<logstash-7-0-0-alpha2,Logstash 7.0.0-alpha2>>
+//* <<logstash-7-0-0-alpha1,Logstash 7.0.0-alpha1>>
+
+Logstash 8.0.0 Release Notes
```

Comment:
I think this is maybe supposed to be a section title?  If not, I think it can be removed